packages/maskbook/src/plugins/Collectible/apis/opensea.ts

```diff
@@ -0,0 +1,114 @@
+import { OpenSeaPort } from 'opensea-js'
+import type { OrderSide, WyvernSchemaName } from 'opensea-js/lib/types'
+import stringify from 'json-stable-stringify'
+import { getChainId } from '../../../extension/background-script/EthereumService'
+import { resolveOpenSeaNetwork } from '../pipes'
+import { OpenSeaBaseURL, OpenSeaGraphQLURL, ReferrerAddress } from '../constants'
+import { Flags } from '../../../utils/flags'
+import type { OpenSeaAssetEventResponse, OpenSeaResponse } from '../UI/types'
+
+const apiKey = 'c38fe2446ee34f919436c32db480a2e3'
```

Comment:
Is it OK?

Reply:
We have many hardcoded API keys that, if added as CI environment variables, will be much better.